app/views/annotation_categories/_annotation_download_modal.html.erb

```diff
@@ -1,32 +1,32 @@
 <%= content_for :modal_id, 'download_dialog' %>
 <%= content_for :modal_title, t('annotations.download.download_annotations') %>
 <%= content_for :modal_content do %>
-
-    <div class="clickable_links">
-      <div class="clickable_text">
-        <img class="clickable_image" src="<%= image_url("icons/download-file.png") %>">
-        <%= t('annotations.download.download_csv') %>
-      </div>
-      <%= link_to '',
-                  { action: 'download',
-                    id: @assignment.id,
-                    format: 'csv' },
-                  onclick: 'modal_download.close();',
-                  class: 'make_div_clickable' %>
-    </div>
-    <div class="clickable_links">
-      <div class="clickable_text">
-        <img class="clickable_image" src="<%= image_url("icons/download-file.png") %>">
-        <%= t('annotations.download.download_yml') %>
-      </div>
-      <%= link_to '',
-                  { action: 'download',
-                    id: @assignment.id,
-                    format: 'yml' },
-                  onclick: 'modal_download.close();',
-                  class: 'make_div_clickable' %>
-    </div>
-    <section class='dialog-actions'>
-      <input type='reset' id="download" value='<%= t(:cancel) %>' onclick='modal_download.close();' />
-    </section>
+	
```

Comment:
Okay, I fixed it using RubyMine's Edit->Convert Indentation->To Spaces. I just pushed it now